spec/helpers/sessions.rb

```diff
@@ -0,0 +1,21 @@
+module SessionHelpers
+
+  def sign_up (user)
+      visit '/users/new'
+      fill_in :name, with: user.name
+      fill_in :username, with: user.username
+      fill_in :email, with: user.email
+      fill_in :password, with: user.password
+      fill_in :password_confirmation, with: user.password_confirmation
+      click_button 'Sign up'
+  end
+
+  def sign_in (user)
```

Comment:
Style/SpaceAfterMethodName: Do not put a space between a method name and the opening parenthesis.